### PR TITLE
Add Istio environment configuration doc page

### PR DESCRIPTION
**Pull request by israel-hdez**
Preview is here: https://deploy-preview-474--kiali.netlify.app/docs/configuration/istio/

Related to kiali/kiali#3858

### CODE REVIEW

**Review by jmazzitelli — APPROVED** [review]
this looks good

File: content/en/docs/Configuration/istio.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
+---
+title: "Istio environment"
```
Comment (jshaughn) [inline content/en/docs/Configuration/istio.md]:
```suggestion
title: "Istio Environment"
```

File: content/en/docs/Configuration/istio.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+recommended labels:
+
+```yaml
+spec:
+  istio_labels:
+    app_label_name: "app"
+    version_label_name: "version"
+```
+
+{{% alert title="Warning" color="warning" %}}
```
Comment (jshaughn) [inline content/en/docs/Configuration/istio.md]:
```suggestion
{{% alert color="warning" %}}
```

File: content/en/docs/Configuration/istio.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+  kiali_feature_flags:
+    clustering:
+      enabled: false
+```
+
+## Root namespace
+
+Istio's _root namespace_ is the namespace where you can create some resources
+to define default Istio configurations and adapt Istio behavior to your
+environment. For more information of this Istio configuration, check the [Istio
```
Comment (jshaughn) [inline content/en/docs/Configuration/istio.md]:
```suggestion
environment. For more information on this Istio configuration, check the [Istio
```

File: content/en/docs/Configuration/istio.md [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+        # Revision string of old Istio version
+        current: "1-10-3"
+        # Revision string of new Istio version
+        upgrade: "1-11-0"
+  kiali_feature_flags:
+    # Turns on canary upgrade support
+    istio_upgrade_action: true
+```
+
+{{% alert title="Warning" color="warning" %}}
```
Comment (jshaughn) [inline content/en/docs/Configuration/istio.md]:
```suggestion
{{% alert color="warning" %}}
```

**Review by jshaughn — CHANGES_REQUESTED** [review]
A couple of minor chnages suggested.  Note that I have been removing the alert titles because the rest of the website does not use them to reinforce the sidebar color.  Titles are reserved for when the information needs to be classified to a certain environment, like `title="OpenShift"`.   This is mainly for readability.  If people think all alerts should have a title, then I'd suggest a PR just for enforcing the new convention, updating all across the website.

**Comment by israel-hdez** [comment]
@jshaughn All your suggestions are in !

**Review by jshaughn — APPROVED** [review]
LGTM!

**Pull request merged**